File: app/src/main/java/com/icesoft/msdb/android/model/Racetrack.java
package com.icesoft.msdb.android.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper=false)
public class Racetrack extends BaseModel {

    private Long id;
    private String name;
    private String location;
    private String countryCode;
    private String logoUrl;
}

File: app/src/main/java/com/icesoft/msdb/android/model/Racetrack.kt
package com.icesoft.msdb.android.model

data class Racetrack(
    val id: Long? = 0,
    val name: String? = null,
    val location: String? = null,
    val countryCode: String? = null,
    val logoUrl: String? = null
): BaseModel()
